### ISSUE TITLE
Switch from pip to pipenv

### ISSUE THREAD

**Issue by palewire** [biglocalnews/civic-scraper#112]

**Comment by zstumgoren** [comment]
@palewire Is this to replace reqs.txt for local dev?

**Comment by palewire** [comment]
Yes, and also for future testing and deployment. Obviously subjective but my take is we'd be better off using the officially recommended Python tool, etc., etc.,

**Comment by zstumgoren** [comment]
Works for me. Fire away whenever you're ready
